curated_transformers/models/config.py

```diff
@@ -1,6 +1,10 @@
 from dataclasses import dataclass
 from typing import Optional
 
+from torch.nn.modules import activation
+
+from curated_transformers.layers.activations import Activation
```

Comment:
Relative import.

Reply:
Fix this and some other absolute imports.